pkg/runtime/service_controller_test.go

```diff
@@ -87,6 +89,12 @@ func TestServiceController(t *testing.T) {
 	sc := ackrt.NewServiceController("bookstore", "bookstore.services.k8s.aws")
 	require.NotNil(sc)
 
+	zapOptions := ctrlrtzap.Options{
+		Development: true,
+		Level:       zapcore.InfoLevel,
+	}
+	fakeLogger := ctrlrtzap.New(ctrlrtzap.UseFlagOptions(&zapOptions))
```

Comment:
Not sure but maybe it's better to pass a fake `io.Writer` to `zapOptions` to prevent cache/controller from logging during unit test phase

Reply:
Maybe. Can do later :)